CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ cross-compilation issue, but will return in v0.13.0.
 - [ENHANCEMENT] Add  `headers` field in `remote_write` config for Tempo. `headers`
   specifies HTTP headers to forward to the remote endpoint. (@alexbiehl)
 
+- [FEATURE] Added Automatic Logging feature for Tempo (@joe-elliott)
```

Comment:
I try to keep these ordered by FEATURE > ENHANCEMENT > BUGFIX > CHANGE, can you move this to the top?